components/emoji/emoji_list_item/emoji_list_item.tsx

```diff
@@ -57,7 +50,7 @@ export default class EmojiListItem extends React.PureComponent {
         } else {
             deleteButton = (
                 <AnyTeamPermissionGate permissions={[Permissions.DELETE_EMOJIS]}>
-                    <AnyTeamPermissionGate permissions={[Permissions.DELETE_OTHERS_EMOJIS]}>
+                    <AnyTeamPermissionGate permissions={[Permissions.DELETE_EMOJIS]}>
```

Comment:
Did you intend to change this permission?

Reply:
Oh, it is my mistake...
I put it back.